SPEC.md

```diff
@@ -75,10 +75,12 @@ This set of formats makes it easy to build, host and secure an ACI using technol
 - Image archives MUST have only two top-level pathnames, `manifest` (a regular file) and `rootfs` (a directory). Image archives with additional files outside of `rootfs` are not valid.
 - All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
 - Image archives MAY be compressed with `gzip`, `bzip2`, or `xz`.
-- Image archives MAY be encrypted with AES symmetric encryption, after (optional) compression/encryption.
+- Image archives MAY be encrypted using PGP symmetric encryption with 256-bit key AES cipher, after optional compression.
```

Comment:
I assume you are referring to the "256-bit key" part. My initial intention was to spell out that the image is encrypted in OpenPGP format, as evidenced by the `gpg` command below this. The "256-bit key" part was added because that same command used it and I had wrongly assumed it to be authoritative. I can remove that part.